main/src/main/java/cgeo/geocaching/utils/ImageUtils.java

```diff
@@ -2,6 +2,7 @@
 
 import cgeo.geocaching.CgeoApplication;
 import cgeo.geocaching.R;
+import cgeo.geocaching.connector.gc.GCParser;
```

Comment:
We should not create a dependency from a general purpose class to a service-specific connector.

Imho the image url conversation logic should be moved to a method inside imageutils, but commented there to be for GC
 That would reflect both that the logic is for for GC only but also reflect that we decided nevertheless to apply it to all images (regardless the source, eg oc or others). 

The last aspect is of utter importance because in the future for all changes we have to make sure that they don't negatively affect images of other service sources. This can't be reached if logic is in GC connector (a developer then has to assume that changes would only affect GC service and nothing else)